coderd/database/queries/insights.sql

```diff
@@ -771,3 +771,18 @@ SELECT
 FROM unique_template_params utp
 JOIN workspace_build_parameters wbp ON (utp.workspace_build_ids @> ARRAY[wbp.workspace_build_id] AND utp.name = wbp.name)
 GROUP BY utp.num, utp.template_ids, utp.name, utp.type, utp.display_name, utp.description, utp.options, wbp.value;
+
+-- name: GetAccumulatedUsersInsights :many
+-- GetAccumulatedUsersInsights returns the accumulated number of users created
+-- in the given timeframe. It returns the accumulated number of users for each date
+-- within the specified timeframe, providing a running total of user sign-ups.
+SELECT
+	date_trunc('day', created_at)::date AS date,
+	COUNT(*) OVER (ORDER BY date_trunc('day', created_at)::date) AS total
+FROM
+	users
+WHERE
+	created_at >= @start_time::timestamptz
+	AND created_at < @end_time::timestamptz
```

Comment:
More for confirmation than a request to change - is it expected to exclude the end_time from the query ? 

If I say I want the insights from 01/01to 02/01- should we include 02/01 ?

Reply:
In-memory DB doesn't match this:

```go
		if user.CreatedAt.Before(arg.StartTime) || user.CreatedAt.After(arg.EndTime) {
```

Missing `|| user.CreatedAt.Equal(arg.EndTime)`.

Reply:
@defelmnq I will double-check that.

@mafredri I didn't get your last comment. Sorry 😞